### PR TITLE
Update fix MintButton bug - Lesson_3_Let_Users_Mint_Your_NFT.md 

### PR DESCRIPTION
**Pull request by k8ives**
This proposed change to the content is to reflect updated code under "Build a Mint NFT button section. The updated code fixes a bug that was allowing users to pass to the DAO member page even if they did not complete the NFT minting process. This change is based on a PR request by another user  [#620 ] - but that PR request had an errant ";" before .catch((err) that caused a syntax error in replit and it wouldn't compile. this change removes the problem ";". Nacho said he would take care of the PR for the actual code in buildspace-dao-final.

### CODE REVIEW

**Comment by smckisson** [comment]
> This proposed change to the content is to reflect updated code under "Build a Mint NFT button section. The updated code fixes a bug that was allowing users to pass to the DAO member page even if they did not complete the NFT minting process. This change is based on a PR request by another user [#620 ] - but that PR request had an errant ";" before .catch((err) that caused a syntax error in replit and it wouldn't compile. this change removes the problem ";". Nacho said he would take care of the PR for the actual code in buildspace-dao-final.

Good catch on the code PR. Thank you! I fixed the compilation error and pulled the logic out into a function to match what was in the project content. Project content had mintNft logic in a function and final project code had it inline. Made them consistent now

**Review by adilanchian — CHANGES_REQUESTED** [review]
Hey @k8ives -

thanks so much for the contribution! Curious - why did you decide to completely rm the `mintNft` function? I feel like we should still keep that in there for now :).

**Comment by k8ives** [comment]
@adilanchian im still a coding newb - i initially saw the solution on discord which had the code inline. i'll admit i was confused by it at the time, but thought maybe there was a reason for it that I didn't understand. in hindsight, i agree it would make more sense to keep the mintNFT function

**Comment by adilanchian** [comment]
Hey @k8ives -

Yo no worries at all! It's awesome you're contributing so thank you!

Ya if you can go ahead and just add your changes to that `mintNFT` function that would be SAWEET.

File: JS_DAO/en/Section_2/Lesson_3_Let_Users_Mint_Your_NFT.md [hunk 1]
```diff
@@ -136,35 +136,36 @@ const App = () => {
...
+    );
+  })
+    .catch((err) => {
+      console.error("failed to claim", err);
+    })
+    .finally(() => {
+      // Stop loading state.
+      setIsClaiming(false);
+    });
+}
```
Comment (adilanchian) [inline JS_DAO/en/Section_2/Lesson_3_Let_Users_Mint_Your_NFT.md:160]: The indention here is just a TAD off :)

Can you correct this for me please?

**Review by adilanchian — CHANGES_REQUESTED** [review]
Almost there!!

Indentation looks a tad off still :)

**Comment by adilanchian** [comment]
Hey @k8ives -

Sorry for the late reply! Everything looks great and will go ahead and merge this in :)

**Comment by adilanchian** [comment]
ALSO - CONGRATS ON YOUR FIRST CONTRIBUTION

🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥

**Comment by adilanchian** [comment]
@all-contributors please add @k8ives  for doc

**Pull request merged**